infrastructure/environments/prod/terragrunt.hcl

```
@@ -0,0 +1,16 @@
+terraform {
+  source = "../../modules/database"
+}
+
+include {
+  path = find_in_parent_folders("root.hcl")
+}
+
+inputs = {
+  node_type         = "DB-PRO2-XXS"
+  volume_size       = 200
```

C'ets l'état actuel, bientôt modifié

```suggestion
  volume_size       = 300
```